### PR TITLE
Update to scala.js 1.x

### PR DESCRIPTION
**Pull request by busti**
I am not certain if it will work like this...  
But there is no example of this in the jitpack docs

### CODE REVIEW

File: .jitpack.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ before_install:
    - wget http://central.maven.org/maven2/org/scala-sbt/sbt-launch/1.2.8/sbt-launch-1.2.8.jar
 install:
    - java -jar sbt-launch-1.2.8.jar +publishM2
+   - SCALAJS_VERSION="1.1.0"
```
Comment (fdietze) [inline .jitpack.yml:9]: I guess you have to put the assignment in the same line as the command...

Comment (busti) [reply .jitpack.yml]: Makes sense. I applied the change in a new commit.  
I will squash those in the next minute or so...

**Comment by busti** [comment]
Okay, I have applied the change and squashed the two commits

File: .jitpack.yml [hunk 2]
```diff
@@ -6,3 +6,4 @@ before_install:
    - wget http://central.maven.org/maven2/org/scala-sbt/sbt-launch/1.2.8/sbt-launch-1.2.8.jar
 install:
    - java -jar sbt-launch-1.2.8.jar +publishM2
+   - SCALAJS_VERSION="1.1.0" ; java -jar sbt-launch-1.2.8.jar +publishM2
```
Comment (fdietze) [inline .jitpack.yml:9]: I mean, without the semicolon. Else, the local variable is not picked up by the java command.

```suggestion
   - SCALAJS_VERSION="1.1.0" java -jar sbt-launch-1.2.8.jar +publishM2
```

**Comment by busti** [comment]
Sorry about that, my bash knowledge is a little rusty.

**Comment by busti** [comment]
Sorry for the spam. Everything should be in order now.  
This PR now depends on https://github.com/fdietze/bench/pull/20 which at this point is already merged.  

This now also includes the following changes:  
* updated maven central url  
* updated scalatestplus to a version supporting scala.js 1.x  
*  updated bench to a version supporting scala.js 1.x

**Comment by busti** [comment]
The travis build failed with the following error:  
```text
[error] (benchJS / Test / scalaJSIR) org.scalajs.core.ir.IRVersionNotSupportedException: Failed to deserialize info of file compiled with Scala.js 1.0 (supported: 0.6.0, 0.6.13, 0.6.3, 0.6.17, 0.6.4, 0.6.6, 0.6.14, 0.6.8, 0.6.30, 0.6.29, 0.6.5, 0.6.15): /home/travis/.coursier/cache/v1/https/jitpack.io/com/github/fdietze/bench/bench_sjs0.6_2.11/5ffab44/bench_sjs0.6_2.11-5ffab44.jar:bench/util/package.sjsir
```

This is a problem in bench, but I do not have the slightest idea what could be causing it. The filename suggests that it was compiled with `scala.js 0.6.x`, but in the message it says that it was compiled with `1.0`.  
Maybe upgrading bench to `1.1` fixes this?

**Comment by fdietze** [comment]
I'm also fine with dropping `0.6` completely and only keeping scalajs `1.0`. That would make things easier, I guess.

**Comment by busti** [comment]
@fdietze Sounds good, should I add that to this PR?

**Comment by fdietze** [comment]
@busti absolutely!

**Comment by busti** [comment]
@fdietze it is done

**Comment by fdietze** [comment]
@busti Thank you! I just gave you the permission to merge yourself.

**Comment by busti** [comment]
I suppose that's a solution. Consider it merged!

**Pull request merged**